### DISCUSSION TITLE
Feature request: Set temp at time

### DISCUSSION THREAD

**Discussion by csmit86** [ScratMan/HASmartThermostat#55] [Ideas]
Say, I wan't my house to be 19,5 degrees at 6 in the morning.
Would be nice if the SmartThermostat could determine when to turn on given the current temp.

**Comment by ScratMan** [comment]
That would be nice, yes, but it would require to include a scheduler inside the thermostat... Not an easy task.

**Comment by nagyrobi** [comment]
See [here](https://community.home-assistant.io/t/smart-heating-scheduler-for-home-assistant-extra-multi-zones-version/237966/1). Usable with this component.
